packages/account-v2/src/modules/src/POI/POI.tsx

```diff
@@ -14,8 +15,10 @@ export const ProofOfIdentity = () => {
     const { isLoading, kyc_auth_status: kycAuthStatus } = useKycAuthStatus();
 
     const poiStatus = kycAuthStatus?.identity.status;
+
     const service = kycAuthStatus?.identity.service;
     const isPOARequired = kycAuthStatus?.address.status === AUTH_STATUS_CODES.NONE;
+    const rejectedReasons = kycAuthStatus?.identity.last_rejected?.rejected_reasons;
```

Comment:
Seems like, `rejectedReasons` value could be `undefined`. In that case, we need to handle that error.

Reply:
Let me check on this